src/client/features/admin/index.js

```diff
@@ -3,12 +3,13 @@ const h = require('react-hyperscript');
 const Link = require('react-router-dom').Link;
 const queryString = require('query-string');
 const _ = require('lodash');
+const classNames = require('classnames');
 
 const Icon = require('../../common/components').Icon;
 const PathwayCommonsService = require('../../services').PathwayCommonsService;
 const CDC = require('../../services/index.js').CDC;
 
-class Admin extends React.Component {
+class Oldadmin extends React.Component {
```

Comment:
@jonahdlin what is the purpose of Oldadmin?

Reply:
My mistake, I meant to remove that. I kept it there so I could test some changes. It serves no purpose.

Reply:
Fix made in this PR and merged